### DISCUSSION TITLE
Station Logbook and Station Location.

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#2762] [Ideas / Feature Requests]
A station location can now be included in multiple station logbooks, which is inconvenient when linking the desired station locations to the log.

It would be nice if the list of possible station locations displayed in the drop-down list of those available for linking to a given station logbook would only show station locations that are not yet bound to any station logbook.

**Comment by phl0** [comment]
Which version are you using? This has been implemented with https://github.com/magicbug/Cloudlog/pull/1570 in August 2022.

**Reply by R1BLH** [reply]
Version 2.5.2 (latest dev branch).
![изображение](https://github.com/magicbug/Cloudlog/assets/16667173/756bb552-3e7c-48a8-844b-ceb0aded83b7)

All other locations are already linked to another station logbook.
![изображение](https://github.com/magicbug/Cloudlog/assets/16667173/f57fbf01-6500-4cda-993f-de2908df58b5)

**Comment by phl0** [comment]
Hmm. That should do. The dropdown contains only station locations which are not yet linked to the Station logbook you are editing.

**Reply by R1BLH** [reply]
Yes, these station locations (which are in the drop-down list) are not linked to the current station logbook under edit. But they are already linked to another station logbook. This is the point.

It would be good to have the possibility to show in the drop-down list only those station locations that are not linked to any station logbook.

**Reply by phl0** [reply]
But that would not allow linking a location to another logbook...

**Reply by R1BLH** [reply]
Yes. 
It is strange for me to link one station location with several station logbooks.
In any case, now i have 76 station locations (thanks to  eQSL,cc) and it is very difficult to find necessary station location when creating new station logbook.
So this could be an option.

**Comment by HB9HIL** [comment]
> Yes. It is strange for me to link one station location with several station logbooks. In any case, now i have 76 station locations (thanks to eQSL,cc) and it is very difficult to find necessary station location when creating new station logbook. So this could be an option.


I don't really understand why so many station locations .. because of eQSL??

**Reply by HB9HIL** [reply]
I have only one Logbook and various Station Locations depending from where I'm operating.

**Reply by R1BLH** [reply]
Yes, because of eQSL. Every time I move to my contryside QTH (typically every weekend in the warm season) I have to create a new profile in eQSL because the gridsquare has changed (in this case the RDA has changed as well).
This is a really annoying situation with eQSL. :(

**Reply by R1BLH** [reply]
I made a second logbook for QSO from another area and DXCC in this case (/0 in the callsign).

**Comment by dg9vh** [comment]
> Yes, because of eQSL. Every time I move to my contryside QTH (typically every weekend in the warm season) I have to create a new profile in eQSL because the gridsquare has changed (in this case the RDA has changed as well). This is a really annoying situation with eQSL. :(

Why not create just one station profile in eQSL that begins at your first QSO and lasts 20 years? You can have parallel valid QTHs in eQSL to one profile... you do not need to create one just for 10 days holiday.

**Reply by R1BLH** [reply]
This is because eQSL requires that valid date periods for each QTH of the same callsign must not overlap.
It would be nice for me have only one profile for one gridsquare (one QTH). :)

**Comment by dg9vh** [comment]
Ah, I had this different in mind but you are right. I tested it on my
account... what a crap :-)

Am Sa., 2. Dez. 2023 um 20:57 Uhr schrieb Michael [R1BLH] <
***@***.***>:

> Why not create just one station profile in eQSL that begins at your first
> QSO and lasts 20 years? You can have parallel valid QTHs in eQSL to one
> profile... you do not need to create one just for 10 days holiday.
>
> This is because eQSL requires that valid date periods for each QTH of the
> same callsign must not overlap.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/magicbug/Cloudlog/discussions/2762#discussioncomment-7739634>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADKN5OV73GWGQOZYLFQQUFLYHOB35AVCNFSM6AAAAABAC3N7D2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TOMZZGYZTI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Amateurfunk ist das, was uns verbindet! Bleiben wir in Kontakt!
DARC e.V., Ortsverband Q03, Neunkirchen
